src/utils/daos_hdlr.c

```diff
@@ -523,8 +523,12 @@ file_close(struct cmd_args_s *ap, struct file_dfs *file_dfs, const char *file)
 }
 
 static int
-file_chmod(struct cmd_args_s *ap, struct file_dfs *file_dfs, const char *path,
-	   mode_t mode)
+file_chmod(struct cmd_args_s *ap,
+	   struct file_dfs *file_dfs,
+	   const char *path,
+	   mode_t mode,
+	   bool ignore_notsup_mod,
+	   uint64_t *num_chmod_enotsup)
```

Comment:
As this is an internal function, I think it's OK to keep the specific parameters especially in light of the inout param used for reporting.